src/AddressUkPafLookupViewBridge.js

```diff
@@ -0,0 +1,139 @@
+var bridge = function (presenterPath) {
+    window.rhubarb.viewBridgeClasses.HtmlViewBridge.apply(this, arguments);
+};
+
+bridge.prototype = new window.rhubarb.viewBridgeClasses.HtmlViewBridge();
+bridge.prototype.constructor = bridge;
+
+bridge.prototype.attachEvents = function() {
+
+    var self = this,
+        alertClass = "c-alert",
+        manualAddressElements = $(".manual-fields"),
```

Comment:
Hi Niall, yeah I agree with you here. The hiding and showing of elements should just be done using standard javascript. If jQuery is needed for some more challenging thing then we could use it to hide and show, but hiding and showing isn't enough of a reason to import a large JS library.

Phil are you happy to make the switch? Do you need me to explain our reasoning further?